tests/testthat/test-individuals.R

```diff
@@ -0,0 +1,105 @@
+test_that("test Create_states with S", {
+
+  population <- squire::get_population("Afghanistan", simple_SEIR = FALSE)
+
+  psq <- squire::parameters_explicit_SEEIR(
+    population = population$n,
+    dt = 1,
+    R0 = 2,
+    tt_contact_matrix = 0,
+    time_period = 1000,
+    contact_matrix_set = squire::contact_matrices[[1]])
+
+  NR <- 0
+  timestep <- 100
+  ind <- 2
+
+  Snew <- individual::State$new("S", sum(psq$S_0))
+
+  states <- create_states(psq)
+
+  expect_equal(Snew$initial_size, states[[1]]$initial_size[1])
+
+})
+
+test_that("test create_states", {
```

Comment:
[In scope]

Reply:
Squire parameters could change and break our state initialisation. Could we test for that?